src/cursor.rs

```diff
@@ -57,6 +57,18 @@ impl Cursor {
                                documents: docs } => {
                 let mut v = VecDeque::new();
 
+                if !docs.is_empty() {
+                    if let Some(&Bson::I32(ref code)) = docs[0].get("code") {                        
+                        // If command doesn't exist or namespace not found, return
+                        // an empty array instead of throwing an error.
+                        if *code == 59 || *code == 26 {
```

Comment:
I don't like the way we are comparing `code` to literal numbers. Within the server, most error codes are defined here: https://github.com/mongodb/mongo/blob/master/src/mongo/base/error_codes.err
so `ErrorCodes::CommandNotFound == 59` and `ErrorCodes::NamespaceNotFound == 26`